test/e2e/kubelet_stats.go

```diff
@@ -157,8 +157,8 @@ func getContainerInfo(c *client.Client, nodeName string, req *kubelet.StatsReque
 		return nil, err
 	}
 	data, err := c.Post().
-		Prefix("proxy").
 		Resource("nodes").
+		SubResource("proxy").
```

Comment:
OK，I'll add version guarded once #18720 merged. And also add for the subresource of service proxy #17493. Just ask a question, why these two PRs couldn't get ack?